scripts/buildnpush.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Helper script to build supported docker images for a specific liboqs/oqsprovider tag/branch and push to docker hub
+# It assumes the user has already successfully logged in to the docker hub namespace used
+
+# set to ascertain docker build cache is not used
+#export NOCACHE="--no-cache"
+
+# docker hub name space to push images to
+export OQS_REPO=openquantumsafe
+
+# docker image tag to set
+export RELEASE_TAG=0.9.2
```

Comment:
I hate to prolong the review process for a script that we may not use again. But... would it be better if these strings were passed as command line arguments, and the script does nothing if all three are not explicitly set?

Reply:
> for a script that we may not use again

That very thought drove my "motivation" to (not) do this in a more parameterized manner. Let's leave this to interested (potential/new) contributors...